cpu/stm32/periph/Makefile

```diff
@@ -49,6 +49,15 @@ ifneq (,$(filter periph_rtt,$(USEMODULE)))
   endif
 endif
 
+# Select the correct implementation for `periph_usbdev`
+ifneq (,$(filter periph_usbdev,$(USEMODULE)))
+  ifneq (,$(filter $(CPU_FAM),f2 f4 f7))
+    SRC += usbdev_otg.c
+  else
+    SRC += usbdev_fs.c  # f0 f1 f3 l0 l1 l4 l5 g4 wb
+  endif
```

Comment:
Since this doesn't work on all L4, I'd be more restrictive for this family here, to avoid confusion. Same for other families, this needs more testing (or references)

Reply:
Indeed, I'll try to restrict it to WB and lower end L4 for now. Can be expand later when we add support for others families then.